molecule/pki/verify.yml

```diff
@@ -44,6 +44,29 @@
     - name: Populate services facts
       ansible.builtin.service_facts:
 
+    - name: Create log files based on journalctl
+      when: ansible_os_family == 'RedHat'
+      ansible.builtin.shell:
+        cmd: journalctl -u {{ item }} > /var/log/alfresco/{{ item }}.log
+      loop:
+        - alfresco-tengine-aio
+        - alfresco-transform-router
+        - alfresco-shared-fs
+      become: true
+
+    - name: Set log files names based on OS family
+      ansible.builtin.set_fact:
+        log_files_names:
+          - file: "{% if ansible_os_family == 'RedHat' %}alfresco-tengine-aio{% else %}ats-ate-aio{% endif %}"
+            pattern: GET Transform Config version
+            edition: Community
+          - file: "{% if ansible_os_family == 'RedHat' %}alfresco-transform-router{% else %}ats-atr{% endif %}"
+            pattern: GET Transform Config version
+            edition: Enterprise
+          - file: "{% if ansible_os_family == 'RedHat' %}alfresco-shared-fs{% else %}ats-shared-fs{% endif %}"
+            pattern: TLS virtual host
+            edition: Enterprise
```

Comment:
oh wait, but you can just make sure that both platforms produced the same logs file by using a more complex loop structure like:

```
- service_name: alfresco-tengine-aio
  file_name: ats-ate-aio
```